packages/perseus/src/perseus-types.ts

```diff
@@ -1059,17 +1060,6 @@ export type PerseusLabelImageWidgetOptions = {
     static: boolean;
 };
 
-export type PerseusLabelImageMarker = {
```

Comment:
@jeremywiebe is particular when it comes to changing `perseus-types.ts`, I would have him sign off first.